e2e/sdk.ERC1155.e2e.test.js

```diff
@@ -0,0 +1,322 @@
+    const sdk = new SDK(acc);
+    const newContract = await sdk.deploy(contractInfo);
+    const loadedContract = await sdk.loadContract({
+      template: TEMPLATES.ERC1155Mintable,
+      contractAddress: newContract.contractAddress,
+    });
+    expect(loadedContract.contractAddress).toEqual(newContract.contractAddress);
+  });
+  it('Load unexisting contract', async () => {
+    const acc = new Auth(authInfo);
+    const sdk = new SDK(acc);
+    const cont = {
+      template: TEMPLATES.ERC1155Mintable,
+      contractAddress: '',
+    };
+    const contract = async () => {
+      await sdk.loadContract(cont);
+    };
+    expect(contract).rejects.toThrow(
+      errorLogger({
+        location: ERROR_LOG.location.SDK_loadContract,
+        message: ERROR_LOG.message.no_address_supplied,
+      }),
+    );
+  });
+  it('Load old contract', async () => {
+    const acc = new Auth(authInfo);
+    const sdk = new SDK(acc);
+    const cont = {
+      template: TEMPLATES.ERC1155Mintable,
+      contractAddress: '0x9F2001302362c94DEaE3a08295a6a957f271F470',
+    };
+    const contract = await sdk.loadContract(cont);
+    expect(contract.contractAddress).toEqual(cont.contractAddress);
+  });
+  it.only('Load new contract and get Metadata', async () => {
+    const acc = new Auth(authInfo);
+    const sdk = new SDK(acc);
+    const newContract = await sdk.deploy(contractInfo);
+    const mintHash1 = await newContract.mint({
+      to: ownerAddress,
+      id: 0,
+      quantity: 1,
+    });
+    const receipt1 = await mintHash1.wait();
+    expect(receipt1.status).toEqual(1);
+    const meta = await sdk.getContractMetadata({ contractAddress: newContract.contractAddress });
+    expect(meta.tokenType).toEqual('ERC1155');
+  });
+
+  it('deploy a contract and addIds', async () => {
+    const acc = new Auth(authInfo);
+    const sdk = new SDK(acc);
+    const newContract = await sdk.deploy(contractInfo);
+    const hash = await newContract.addIds({ ids: [0] });
+    const receipt1 = await hash.wait();
+    expect(receipt1.status).toEqual(1);
+    const mintHash = await newContract.mint({
+      to: ownerAddress,
+      id: 0,
+      quantity: 1,
+    });
+    await mintHash.wait();
+    await wait(
+      async () => {
+        const response = await nftApiClient.getAllNfsFromCollection(newContract.contractAddress);
+        return response.data.total === 1;
+      },
+      90000,
+      1000,
+      'Waiting for NFT collection to be available',
+    );
+    const collection = await nftApiClient.getAllNfsFromCollection(newContract.contractAddress);
+    expect(collection.data.total).toEqual(1);
+  });
+  it.skip('deploy a contract and setBaseURI', async () => {
+    const acc = new Auth(authInfo);
+    const sdk = new SDK(acc);
+    const newContract = await sdk.deploy(contractInfo);
+    const hash = await newContract.setBaseURI({
+      baseURI: 'https://ipfs.io/ipfs/QmeSjSinHpPnmXmspMjwiXyN6zS4E9zccariGR3jxcaWtq',
```

Comment:
We should not use an IPFS Gateway in the baseURI/tokenURI. Please use the IPFS URL: `ipfs://QmeSjSinHpPnmXmspMjwiXyN6zS4E9zccariGR3jxcaWtq`